### PR TITLE
RR-775 - Submit Additional Training through to Have You Worked Before

### PR DESCRIPTION
**Pull request by nathanrussell-moj-digital**
This PR implements the page submission from Additional Training through to Have You Worked Before

### CODE REVIEW

File: server/routes/induction/update/additionalTrainingUpdateController.test.ts [hunk 1]
```diff
@@ -294,12 +272,8 @@ describe('additionalTrainingUpdateController', () => {
       expect(req.session.pageFlowHistory).toEqual(expectedPageFlowHistory)
     })
 
-    it('should update InductionDto and redirect to In Prison Work view given there is an updateInductionQuestionSet on the session', async () => {
+    it('should update InductionDto and redirect to In Prison Work view given shprt question set journey', async () => {
```
Comment (leejacobson-moj) [inline server/routes/induction/update/additionalTrainingUpdateController.test.ts:275]: Minor typo:
```suggestion
    it('should update InductionDto and redirect to In Prison Work view given short question set journey', async () => {
```

File: server/routes/induction/create/additionalTrainingCreateController.test.ts [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+
+      // Then
+      const updatedInductionDto = req.session.inductionDto
+      expect(updatedInductionDto.previousTraining.trainingTypes).toEqual(expectedUpdatedAdditionalTraining)
+      expect(updatedInductionDto.previousTraining.trainingTypeOther).toEqual(expectedUpdatedAdditionalTrainingOther)
+      expect(res.redirect).toHaveBeenCalledWith(expectedNextPage)
+      expect(req.session.additionalTrainingForm).toBeUndefined()
+    })
+
+    it('should update InductionDto and redirect to In Prison Work view given shprt question set journey', async () => {
```
Comment (leejacobson-moj) [inline server/routes/induction/create/additionalTrainingCreateController.test.ts:194]: Minor typo:
```suggestion
    it('should update InductionDto and redirect to In Prison Work view given short question set journey', async () => {
```

**Review by leejacobson-moj — CHANGES_REQUESTED** [review]
Looks good. Noticed a couple of typos. Will approve as soon as those are updated. 🙂

File: server/routes/induction/update/additionalTrainingUpdateController.test.ts (hunk 1 above)
Comment (nathanrussell-moj-digital) [reply server/routes/induction/update/additionalTrainingUpdateController.test.ts:275]: Good spot; fixed 👍

File: server/routes/induction/create/additionalTrainingCreateController.test.ts (hunk 2 above)
Comment (nathanrussell-moj-digital) [reply server/routes/induction/create/additionalTrainingCreateController.test.ts:194]: Fixed 👍

**Review by leejacobson-moj — APPROVED** [review]
Looks good. Approved 👍

**Pull request merged**